test/Rating.js

```diff
@@ -1,9 +1,138 @@
 'use strict';
 
+import dom from 'metal-dom';
 import Rating from '../src/Rating';
 
 describe('Rating', function() {
-	it('should be tested', function() {
-		assert.fail('No tests for this module yet.');
+	var rating;
+
+	afterEach(function() {
+		rating.dispose();
+	});
+
+	it('should render with default attributes.', function() {
+		rating = new Rating().render();
+		assert.strictEqual('', rating.element.style.display);
+	});
+
+	it('should render specified label.', function() {
+		rating = new Rating({
+			label: "This is an awesome Metal Component"
+		}).render();
+		assert.strictEqual("This is an awesome Metal Component", rating.element.textContent);
+	});
+
+	it('should highlight until item clicked', function() {
+		rating = new Rating().render();
+
+		var elements = rating.element.querySelectorAll('.rating-item');
+		dom.triggerEvent(elements[2], 'click');
+
+		rating.once('stateSynced', function(){
```

Comment:
Since `stateSynced` events are asynchronous you need to use the `done` function that karma passes to the test functions, and call it inside the `stateSynced` callback function, otherwise the asserts will be skipped as the test will be assumed to be synchronous.